### DISCUSSION TITLE
Random order does not work

### DISCUSSION THREAD

**Discussion by MrGreen68** [nolimits4web/swiper#5870] [Q&A]
Is there any way to randomly display the Caroussel items?

When I place the random images from my JavaScript array in the Swiper container, nothing appears.
If I place them outside the Swiper container everything is OK.

Thanks for any hints in advance!

**Comment by osopolar** [comment]
You may randomize the slides before initializing the swiper. An example for how to initialize an array you may find at:

https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array/2450976#2450976

I did it like:

    const $parent = $(".swiper-wrapper");
    const $children = $parent.children();
    $parent.empty().append(shuffle($children));
